leases/lease.go

```diff
@@ -0,0 +1,50 @@
+package leases
+
+import (
+	"encoding/json"
+	"time"
+)
+
+const (
+	TimeFormat = time.RFC3339
```

Comment:
Just a thought: it would be nice if all deis go projects that speak a common idiom had a common time format. So something like `TimeFormat = deisConstantLibrary.TimeFormat`